elasticsearch_dsl/_async/search.py

```diff
@@ -175,6 +185,10 @@ class AsyncMultiSearch(MultiSearchBase[_R]):
 
     _using: AsyncUsingType
 
+    if TYPE_CHECKING:
+
+        def add(self, search: AsyncSearch[_R]) -> Self: ...  # type: ignore[override]
```

Comment:
There are a few methods that override their definition in the parent class. I have type ignored them for now, because at some point I want to revisit them and see if they can be changed to match.